### DISCUSSION TITLE
Default Values won't work for Customer Portal

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#16873] [Ignored/Incomplete]
Hi all. My company is looking to utilize the YetiForce Customer Portal. However, when trying to edit values for tickets, such as a default value for who the ticket should be assigned to, it does not work. In this case, my company wants the default Assigned To option to be set to our Support Group, which handles tickets created by customers. I was able to easily edit this by going into Software Configuration > Standard Modules > Edit Fields > portal (Webservice Premium) and then changed the Assigned To values to Visibility: Field disabled everywhere so that the customer doesn't see who it is assigned to, and Default value: checked with the value set to Support Group. These settings save and when I go back to the field, it displays these settings, confirming they were saved. However, when I go to create a ticket via the portal using a test account I made, it still says "Assigned To: Test User", which is the user that is logged in. I know by default tickets are assigned to the user creating them, but I had updated this to Support Group in the fields and it shows that those settings saved, but do not seem to work when creating tickets in the portal.

I would appreciate any and all help to fix this! It seems to be a clear bug somewhere in the actual code of the system.
Thanks in advance!

**Comment by mariuszkrzaczkowski** [comment]
Attach screenshots showing the problem because I don't understand what exactly the problem is, this functionality works for our customers, you probably need to configure something else

**Reply by bfeddes** [reply]
The problem is that when I edit the default values for the portal, they save and shows that I made the value default, but when I then make a ticket within the portal, the default values are not applied. I attached two screenshots showing the issue. 

This screenshot shows the settings I have configured for tickets in the portal, as you can see, I want the default **assigned to** value to be Support Group. I changed this setting by going to: **Software configuration > Standard Modules > Edit fields > portal (Webservice Premium) > Tickets > Assigned To > Set default value to Support Group.**
![fieldview](https://user-images.githubusercontent.com/112420814/218174234-a9e318db-fc16-4b1e-8c22-5fc43484832c.PNG)

This next screenshot then shows that the default value does not apply to creating tickets in the portal. The default value saves and when I go back to the field, it shows that the default value is still Support Group. However, the configuration does not seem to apply to the portal when trying to create the ticket. The screenshot below shows the default value when creating a ticket in the portal. Instead of saying **Administrator Urbancom (Help Desk Department)** it should be saying **Support Group**. I do not know why it doesn't though. I can change the Assigned To field to Support Group, but have to do so manually which is not what I want our customers to be able to do. I want all created tickets to be automatically assigned to the Support Group.
![issuee](https://user-images.githubusercontent.com/112420814/218174694-3599c103-675a-4013-98ed-a59d4ae3f5b2.PNG)

**Reply by mariuszkrzaczkowski** [reply]
create an issue with a set of information that will allow you to trigger the bug

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13908
